db/db_impl/db_impl_files.cc

```diff
@@ -1023,4 +997,47 @@ Status DBImpl::DeleteUnreferencedSstFiles(RecoveryContext* recovery_ctx) {
   return s;
 }
 
+void DBImpl::TrackExistingDataFiles(
```

Comment:
Nit: Maybe move this to db_impl_open.cc since its only called by `DBImpl::Open`?

Reply:
That's a good call. Thank you for the suggestion!